ibm/resource_ibm_atracker_target.go

```diff
@@ -0,0 +1,238 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/atrackerv1"
+)
+
+func resourceIBMAtrackerTarget() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMAtrackerTargetCreate,
+		ReadContext:   resourceIBMAtrackerTargetRead,
+		UpdateContext: resourceIBMAtrackerTargetUpdate,
+		DeleteContext: resourceIBMAtrackerTargetDelete,
+		Importer:      &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Update: schema.DefaultTimeout(20 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The name of the target. Must be 256 characters or less.",
+			},
+			"target_type": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: InvokeValidator("ibm_atracker_target", "target_type"),
+				Description:  "The type of the target.",
+			},
+			"cos_endpoint": &schema.Schema{
+				Type:        schema.TypeList,
+				MinItems:    1,
+				MaxItems:    1,
+				Required:    true,
+				Description: "Property values for a Cloud Object Storage Endpoint.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The host name of this COS endpoint.",
+						},
+						"target_crn": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The CRN of this COS instance.",
+						},
+						"bucket": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "The bucket name under this COS instance.",
+						},
+						"api_key": &schema.Schema{
+							Type:        schema.TypeString,
+							Required:    true,
```

Comment:
Add sensitive true

Reply:
We need to add `x-terraform-sensitive: true` to this property in the API definition.  That will cause the generator to add `Sensitive: true,` here.